hardware/raspberrypi/dpi/README.md

```diff
@@ -1,6 +1,6 @@
 # DPI (Parallel Display Interface)
 
-An up-to-24-bit parallel RGB interface is available on all Raspberry Pi boards with the 40 way header (A+, B+, Pi2, Pi3, Zero) and Compute Module. This interface allows parallel RGB displays to be attached to the Raspberry Pi GPIO either in RGB24 (8 bits for red, green and blue) or RGB666 (6 bits per colour) or RGB565 (5 bits red, 6 green, and 5 blue).
+An up-to-24-bit parallel RGB interface is available on all Raspberry Pi boards with the 40 way header (A+, B+, Pi2, Pi3, Pi4, Zero) and Compute Modules. This interface allows parallel RGB displays to be attached to the Raspberry Pi GPIO either in RGB24 (8 bits for red, green and blue) or RGB666 (6 bits per colour) or RGB565 (5 bits red, 6 green, and 5 blue).
```

Comment:
@JamesH65 Perhaps it's worth changing "with the 40 way header (A+, B+, Pi2, Pi3, Pi4, Zero) and Compute Modules." to just "with the 40 way header, and the Compute Modules." ?